src/cli/config-flags.ts

```diff
@@ -193,6 +207,7 @@ export type StringCLIFlag = ArrayValuesAsUnion<typeof STRING_CLI_FLAGS>;
 export type StringArrayCLIFlag = ArrayValuesAsUnion<typeof STRING_ARRAY_CLI_FLAGS>;
 export type NumberCLIFlag = ArrayValuesAsUnion<typeof NUMBER_CLI_FLAGS>;
 export type StringNumberCLIFlag = ArrayValuesAsUnion<typeof STRING_NUMBER_CLI_FLAGS>;
+type BooleanStringCLIFlag = ArrayValuesAsUnion<typeof BOOLEAN_STRING_CLI_FLAGS>;
```

Comment:
I don't believe we need to export these anymore, but please LMK if that understanding is incorrect

Reply:
yeah I don't see any references to them outside of the `config-flags` module - possibly we can stop exporting all of them. I looked back on the PR where they were introduced and I think it was just a 'I guess we should export these?' sort of moment